custom_components/tuya_local/devices/tera_w01_evcharger.yaml

```diff
@@ -0,0 +1,270 @@
+name: Tera W01 EV charger

```

Comment:
This top level name should be without product-specific branding (eg, just "EV charger")

Reply:
Sure!